helpers/form/ElementMapFactory.php

```diff
@@ -286,15 +289,14 @@ private function getDescriptionFromTranslatedPropertyLabel(
         core_kernel_classes_Property $property,
         string $language
     ) {
-        $propertyLabel = current($property->getPropertyValues(
-            new core_kernel_classes_Property(OntologyRdfs::RDFS_LABEL),
-            [
-                'lg' => $language,
-                'one' => true
-            ]
-        ));
-
-        if (empty(trim($propertyLabel))) {
+        $propertyLabel = current(
+            $property->getPropertyValues(
+                $this->getProperty(OntologyRdfs::RDFS_LABEL),
+                ['lg' => $language, 'one' => true]
+            )
+        );
+
+        if (false !== $propertyLabel && empty(trim($propertyLabel))) {
```

Comment:
Maybe, to be more precise.

```suggestion
        if (is_string($propertyLabel) && empty(trim($propertyLabel))) {
```

Reply:
The issue is that `current` returns false if the array is empty.